### PR TITLE
feat : add more apis to the list

### PR DESCRIPTION
**Pull request by thecasanovaa**

### CODE REVIEW

File: src/content/apis/images/giphy-api.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: GIPHY API
+description: GIPHY's API makes it dead simple for developers to incorporate GIPHY's vast library right inside of their apps.
+categories:
+  - images
+features:
+  http: true
```
Comment (TheLearneer) [inline src/content/apis/images/giphy-api.md:7]:
```suggestion
  https: true
```

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Please make the change, and we are good to go.

**Comment by thecasanovaa** [comment]
done sir

**Review by TheLearneer — APPROVED** [review]
LGTM

**Pull request merged**